frontera/utils/heap.py

```diff
@@ -65,3 +65,6 @@ def _extract_object(self):
             return wrapper.obj
         except IndexError:
             return None
+
+    def __len__(self):
```

Comment:
yes, it is. It is actually part of what we need in HCF middleware for controlling the next requests logic